src/vopr_hub/SETUP.md

```diff
@@ -120,7 +120,7 @@ export CURRENT_VOPR=$1
 go run ./src/vopr_hub/scheduler/main.go
 
 # Run the VOPR a few times before we go back to the top:
-zig/zig run ./src/vopr.zig -- --send="<address>" --simulations=5
+zig/zig run ./src/vopr.zig -- --send="<address>" --simulations=100
```

Comment:
This helps avoid running into rate limits from github. It is already running on our voprhub instance.

Reply:
(It's unrelated to state sync... just included in this PR for convenience.)